api/prisma/migrations/00_init/migration.sql

```diff
@@ -1292,4 +1292,11 @@ ALTER TABLE
 ADD
       CONSTRAINT "FK_87b8888186ca9769c960e926870" FOREIGN KEY ("user_id") REFERENCES "user_accounts"("id") ON
 DELETE CASCADE ON
-UPDATE CASCADE;
\ No newline at end of file
+UPDATE CASCADE;
```

Comment:
Are we able to update this migration as it has already run?

Reply:
So the change here is to give us translations locally and on the core environment whenever we reseed (`yarn setup`) 

we already have those migrations in HBA prod, so we don't want to insert that there


so I added a generalized version of HBA's translations for us here. We can run just that part separately on the core environment for now and then whenever we reseed there we'd have translations again